src/main/java/com/puppycrawl/tools/checkstyle/checks/coding/UnnecessaryParenthesesCheck.java

```diff
@@ -479,7 +479,7 @@ public void visitToken(DetailAST ast) {
         final int type = ast.getType();
         final DetailAST parent = ast.getParent();
 
-        if (type == TokenTypes.LAMBDA && isLambdaSingleParameterSurrounded(ast)) {
+        if (isLambdaSingleParameterSurrounded(ast)) {
```

Comment:
Please update javadoc of method, since we previously assumed that all tokens checked by this method were of type `LAMBDA`.

Reply:
Done.